cmd/minikube/cmd/image.go

```diff
@@ -310,6 +311,12 @@ var buildImageCmd = &cobra.Command{
 				// Otherwise, assume it's a tar
 			}
 		}
+		if runtime.GOOS == "windows" && strings.Contains(dockerFile, "\\") {
+			// if dockerFile is a DOS path, and user did not explicitly forbid this conversion, translate it into UNIX path
```

Comment:
```suggestion
			// if dockerFile is a DOS path, translate it into UNIX path
```
With the changes the user can no longer forbid this, so comment is incorrect